lib/ssl.js

```diff
@@ -32,7 +33,7 @@ function generateCertificate(outfile, service_name, cb) {
         '/usr/bin/openssl req -x509 -nodes -days 365 ' +
             '-newkey rsa:2048 -keyout %s -out %s ' +
             '-config /opt/local/etc/openssl/openssl.cnf ' +
-            '-subj "/C=US/ST=CA/O=Joyent/OU=manta/CN=%s"',
+            '-subj "/C=US/ST=CA/O=Triton Data Center/OU=manta/CN=%s"',
```

Comment:
Does "Subject name change" merit a package.json bump?

Reply:
Probably should, yeah.